TU-Agent/agentKnowledge.pl

```diff
@@ -30,6 +31,20 @@
 :- dynamic indicator/4.
 :- dynamic indicatorGoal/2.
 
+
+%Knowledge for answering a request
+answerRequest(Category, PopupID) :- requestAnswered(Category, PopupID).
+empty(X) :- length(X, 0).
+
+%The agent finds a price to be acceptable when the offered price is at least 50 euro higher than the ground price for offices in Delft (252)
+acceptablePrice(Price, Areasize) :- Areasize * 252 + 50 < Price.
+
+%this believe ensures that indicatorlink gets generated only once
+%it gets deleted after indicatorlink is inserted as believe
+readIndicatorlink.
```

Comment:
readIndicatorlink has been removed since we refactored the init module. So this isn't necessary.